gitgraph

```diff
@@ -252,6 +255,8 @@ function main()
         then
             dia "$TMP_SVG"
             rm "$TMP_SVG"
+        else
```

Comment:
Something to think about. Just do this if user request with an output file. Say:

- Add '--output' option.
- If output option is present pipe command with > to that file

relates with #1